components/ai_chat/resources/page/stories/story_utils/locale.ts

```diff
@@ -118,5 +118,6 @@ provideStrings({
   menuTitleCustomModels: 'Custom models',
   startConversationLabel: 'Start new conversation',
   goBackToActiveConversationButton: 'Go back to the active conversation',
-  conversationListUntitled: 'New conversation'
+  conversationListUntitled: 'New conversation',
+  stopGenerationButtonLabel: 'Stop generation',
```

Comment:
Should this be `Stop answering` to match whats in strings?
``` suggestion
  stopGenerationButtonLabel: 'Stop answering',
```

Reply:
Fixed https://github.com/brave/brave-core/commit/48837395ec37f173b567ace0661aeb30de87d49d#diff-f7b91893b223a6a6d056355c6183cac1e6af0aa4bed16b0485bc77fc89e01342R122 thanks